scripts/set-license-text.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# This script tries to figure out what year a package was created and updates
+# its `LICENSE` file accordingly (including copyright owner)
+
+set -euo pipefail
+
+declare -A years
+for pkg in ./packages/*; do
+  year=$(git --no-pager log --diff-filter=A --follow --format=%ad --date format:%Y --reverse -- "${pkg}" | head -n1) || true
+  years[$pkg]=$year
+done
+
+for i in "${!years[@]}"; do
+  license="${i}/LICENSE"
+  year="${years[$i]}"
+  if [ -f "$license" ]; then
+    sed -i '' -e "s/\[yyyy\]/${year}/g; s/\[name\ of\ copyright\ owner\]/Endo Contributors/g" "$license"
```

Comment:
I would be delighted to have this in `scripts/create-package.sh` but I can wait until the next package gets created to make that update.

Reply:
Might as well do it now, because whoever makes the next package may not be one of us.

I combined this into a single search/replace operation.